### ISSUE TITLE
 ACT ff89c9: fix `aria_parent_required` WCAG mapping from 4.1.2 to 1.3.1

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1956]
`aria_parent_required` should not map to WCAG 4.1.2, but to 1.3.1 to match ACT `ff89c9`

- [x] also change the Help to list 1.3.1, not 4.1.2

The ACT rule maps to WCAG requirement:
1.3.1: Info and Relationships (Level A)

### Supporting notes and details
 [ACT Master 2024.xlsx](https://ibm.box.com/s/evec4kfixgzjw8tmghnr9f13xxw99e33)

**Comment by philljenkins** [comment]
mapping change coded in phill-ACT-mapping
changed 4.1.2 to 1.3.1 in both rule and Help reference

**Comment by philljenkins** [comment]
Help formats correctly
and references 1.3.1
 
<img width="1728" alt="Screenshot 2024-07-26 at 12 47 29 PM" src="https://github.com/user-attachments/assets/f29bbbca-f432-430f-930d-7a84ae9fb691">

**Comment by philljenkins** [comment]
correct in Rule-set

<img width="1011" alt="Screenshot 2024-07-26 at 1 04 03 PM" src="https://github.com/user-attachments/assets/0c05ff37-8e10-4d37-b3ad-00af0c6039cc">

**Comment by philljenkins** [comment]
Reports correctly against ACT test cases bc4a75 in [act-report-v2 2024.txt](https://ibm.box.com/s/or6i901ke3etyv5xoo5e61yynx6xem77):

> ARIA required context role (https://www.w3.org/WAI/standards-guidelines/act/rules/ff89c9)
>  + Passed Example 1: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/3ae3bc1c993acb6baaad2811cbd6139a8093361c.html
>  + Passed Example 2: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/44afe364fc9417fd5663599145f670552f507ab0.html
>  + Passed Example 3: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/694b790e4f1eae0f22aef2e7c06b646b25db8e1d.html
>  + Passed Example 4: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/b81cf2923d30381d48980be59729a5cb0d792059.html
>  + Passed Example 5: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/2ffe7d6cfa547dc8b107922a6bd7542ea36c96d6.html
>  + Passed Example 6: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/1acc47f25d4931c25fe3efbb676af6fd4e2ee57e.html
>  + Failed Example 1: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/cd55d1d52c286ac6b342155dde8fcfa49c82ae4a.html
>  + Failed Example 2: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/2fb70cb7f44a01a2d75f4ef7ca7992cf3fb4fe1d.html
>  + Failed Example 3: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/52508dc0ac389108301d7cbd7f931be45a45741f.html
>  + Failed Example 4: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/f8e3dbe601969ab54954447e04ae384eb52d7082.html
>  + Inapplicable Example 1: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/9f86cf6493bf2315ce01cec636014d1c059d6581.html
>  + Inapplicable Example 2: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/7ec257f7f32bbe21231743ef1da46943584142c8.html
>  + Inapplicable Example 3: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/a582209de4a1d8ed76f54ca2e1f76d1efdbd499e.html
>  + Inapplicable Example 4: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/3457868b79bad5b8cf2320c88cd5f542f9388cda.html
>  + Inapplicable Example 5: https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/ff89c9/48dc663078fb5421332814b72bd0079f90aad09a.html
